package.json

```diff
@@ -47,6 +47,7 @@
     "zuul": "^3.7.2"
   },
   "dependencies": {
-    "async": "^1.5.0"
+    "async": "^1.5.0",
```

Comment:
Yes, I left everything so that we could easily test before/after in one place if needed. Will remove soon.